EXAMPLES.md

```diff
@@ -325,30 +338,33 @@ ShopifyBuy.Client().products((products, error) => {
 
 ```
 
+#### Additional Build Settings
 
-**Additional Build Settings**
-
-To use Apple Pay, you must also enable `Background fetch` in the Player Settings.
+To use Apple Pay, you must also enable **Background fetch** in the Player Settings:
 
-1. Open Player Settings, Edit -> Project Settings -> Player
-2. Select `Settings for iOS`
-3. Under `Other Settings`, set `Behaviour in Background` to `Custom`
-4. Enable `Background fetch`
+1. Open Player Settings, and click on *Edit* > *Project Settings* > *Player*.
+2. Select **Settings for iOS**.
+3. Under **Other Settings**, set **Behavior in Background** to **Custom**.
+4. Enable **Background fetch**.
 
-**Enabling Apple Pay on your Store**
+#### Enabling Apple Pay on your Store
 
-To enable Apple Pay for your store through an app, follow the following steps:
+To enable Apple Pay for your store through an app:
 
 1. Add the [Mobile App sales channel](https://help.shopify.com/api/sdks/custom-storefront/mobile-buy-sdk/add-mobile-app-sales-channel) in your Shopify admin.
 2. Enable Apple Pay in the Mobile App settings page.
 
-**Notes**
+#### Notes
 
 `CheckoutWithNativePay` will throw an exception if the device is unable to make a payment through Apple Pay. So it is essential that `CanCheckoutWithNativePay` is used.
 
-**Errors**
+#### Errors
+
+On failure, you will receive a `ShopifyError`. There are 3 types of `ShopifyError` that you will have to handle:
 
-On failure, you will receive a `ShopifyError`. The types of `ShopifyError` you will have to handle are `HTTP`, `NativePaymentProcessingError`, and `GraphQL`.
+* `HTTP`
```

Comment:
It would be valuable. It could be:

```
`HTTP` errors will be thrown when there was an issue connecting or downloading to a web server in the back end. 
```